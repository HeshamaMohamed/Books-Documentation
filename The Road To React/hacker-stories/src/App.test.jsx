import { describe, it, expect } from 'vitest';

describe('something truthy and falsy', () => {
  it('true to be true', () => {
    expect(true).toBe(true);
    expect(true).toBeTruthy();
  });

  it('false to be falsy', () => {
    expect(false).toBe(false);
    expect(false).toBeFalsy();
  });
});
