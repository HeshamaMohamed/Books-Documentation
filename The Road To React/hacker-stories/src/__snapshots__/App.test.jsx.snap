// Vitest Snapshot v1

exports[`SearchForm > renders snapshop 1`] = `
<form>
  <label
    for="search"
  >
    <strong>
      Search: 
    </strong>
  </label>
  <input
    id="search"
    type="text"
    value="React"
  />
  <button
    type="submit"
  >
    Submit
  </button>
</form>
`;
